modules/microshift-install-multus-rpm.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-multus_{context}"]
+== Installing the multiple networks plugin
+
+Use this procedure to install the {microshift-short} Multus CNI plugin alongside a new {microshift-short} installation. The {microshift-short} Multus Container Network Interface (CNI) plugin is not installed by default. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package.
+
+[IMPORTANT]
+====
+The Multus RPM cannot be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-multus
```

Comment:
So, I think if it's just `dnf install microshift-multus` then we need to restart CRIO... We don't if we do `dnf install microshift microshift-multus`.

Reason is: in %post section in the microshift RPM we restart CRIO, but there's nothing like this in `microshift-multus`.

Let's discuss if we should change it for UX sake.